tests/Ardalis.HttpClientTestExtensions.Core/Entities/Country.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Ardalis.HttpClientTestExtensions.SharedKernel;
+using Ardalis.HttpClientTestExtensions.SharedKernel.Interfaces;
+
+namespace Ardalis.HttpClientTestExtensions.Core.Entities;
+
+public class Country : BaseEntity<string>, IAggregateRoot
+{
+  public string Name { get; set; } = string.Empty;
+  public virtual List<City>? Cities { get; set; }
```

Comment:
Probably good to always initialize list properties in their declarations

Reply:
Done